readthedocs/proxito/tests/test_redirects.py

```diff
@@ -307,6 +405,8 @@ def test_http_public_domain_https_redirect(self):
             r["Location"],
             "http://project.dev.readthedocs.io/en/latest/",
         )
+        self.assertEqual(r.headers["CDN-Cache-Control"], "public")
```

Comment:
Shouldn't we have some tests for `private` here as well?

Reply:
All redirects are cached, we don't have redirects that would be private.